File: csharp/SEP3UI/Model/Order.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace SEP3UI.Model {
    public class Order {
        public IList<Item> Items { get; set; }
        
        public User User { get; set; }
        public int Id { get; set; }
        
        public Address Address { get; set; }
        
        public DateTime DateTime { get; set; }
        
        [JsonConverter(typeof(JsonStringEnumConverter))]
        public OrderStatus OrderStatus { get; set; }
    }
}

File: java/src/main/java/database/model/Order.java
package database.model;

import database.model.enums.OrderStatus;

import java.util.List;

public class Order {
    private List<Item> items;
    private User user;
    private int id;
    private Address address;
    private MyDateTime datetime;
    private OrderStatus orderStatus;

    public Order(List<Item> items, User user, int id, Address address, MyDateTime datetime, OrderStatus orderStatus) {
        this.items = items;
        this.user = user;
        this.id = id;
        this.address = address;
        this.datetime = datetime;
        this.orderStatus = orderStatus;
    }

    public List<Item> getItems() {
        return items;
    }

    public User getUser() {
        return user;
    }

    public int getId() {
        return id;
    }

    public Address getAddress() {
        return address;
    }

    public MyDateTime getDatetime() {
        return datetime;
    }

    public OrderStatus getOrderStatus() {
        return orderStatus;
    }
}
